changelog/v0.24.9/13501-report-missing-images-in-scan.yaml

```diff
@@ -0,0 +1,9 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo-mesh-enterprise/issues/13501
```

Comment:
I think this is actually from a different issue: https://github.com/solo-io/gloo-mesh-enterprise/issues/13499#issuecomment-1847675824

But does not resolve that issue, just the issue comment

Reply:
Good catch, thank you!